fields/acf-vimeo_pro_data-v5.php

```diff
@@ -1,33 +1,26 @@
 <?php
 
-// exit if accessed directly
-if (! defined('ABSPATH')) {
```

Comment:
Is this bad ?

Reply:
I don't think so.... it wasn't included on https://github.com/AdvancedCustomFields/acf/blob/master/includes/fields/class-acf-field-text.php which is what I updated the structure with so I removed it.